db/migrate/20210825063235_dropusers.rb

```diff
@@ -0,0 +1,5 @@
+class Dropusers < ActiveRecord::Migration[6.1]
+  def change
+    drop_table :users
```

Comment:
why you need to drop the table users when you even create it?

Reply:
i also don't understand.... so what should i do ? :D

Reply:
You should delete this file, it make you code confuse